test/support/commands.js

```diff
@@ -40,7 +40,6 @@ Cypress.Commands.add('routesForPatientDashboard', () => {
     .routePatient()
     .routePatientActions()
     .routePatientFlows()
-    .routePatientField()
```

Comment:
possibly we can remove `routePatientField`'s support/api method.  I don't think we should be using it now?

Reply:
It's currently used in this form test: https://github.com/RoundingWell/care-ops-frontend/blob/develop/test/integration/forms/form.js#L275.

So I'm not really sure. That's why I left it originally.

Reply:
ah yep.. I think we exposed the form to be able to request and update a field async